manifests/bond-cni.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: bond-cni
+  labels:
+    tier: node
+    app: bond-cni
+spec:
+  selector:
+    matchLabels:
+      app: bond-cni
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 10%
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: bond-cni
+    spec:
+      nodeSelector:
+        kubernetes.io/arch: amd64
+        kubernetes.io/os: linux
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: bond-cni-plugin
+        image: ghcr.io/k8snetworkplumbingwg/bond-cni:latest
+        resources:
+          requests:
+            cpu: "10m"
+            memory: "15Mi"
+        volumeMounts:
+        - name: cnibin
+          mountPath: /host/opt/cni/bin/
+      volumes:
+        - name: cnibin
+          hostPath:
+            path: /host/opt/cni/bin/
```

Comment:
This one doesn't see right the hostPath should be opt/cni/bin and inside the container we should have `/host/opt/cni/bin/` no?